packages/calypso-e2e/src/lib/flows/gutenboarding-flow.ts

```diff
@@ -68,10 +70,19 @@ export class GutenboardingFlow {
 	 * @param {string} text User-visible text on the button.
 	 * @returns {Promise<void>} No return value.
 	 */
-	async clickButton( text: string ) {
+	async clickButton( text: string ): Promise< void > {
 		await this.page.click( selectors.button( text ) );
 	}
 
+	/**
+	 * Clicks on the WP Logo on top left.
+	 */
+	async clickWpLogo(): Promise< void > {
+		await Promise.all( [ this.page.waitForNavigation(), this.page.click( selectors.wpLogo ) ] );
+
+		await this.page.waitForLoadState( 'networkidle' );
```

Comment:
I'd make sure this is really necessary here! In my experience, it can lead to unnecessarily long wait times as there is something that's still loading that shouldn't really stop the test from moving on.

Reply:
Originally, this was a `waitForLoadState('load')` but that was not enough of a wait; the pending navigations ended up swallowing the click on the navbar.

Reply:
I tested some more and the results seem to indicate the standard `load` state would suffice, and that's already a part of the `NavbarComponent.clickMe` method that constitutes the `CloseAccountFlow`.